sqlglot/lineage.py

```diff
@@ -277,7 +281,7 @@ def to_node(
 
     for c in source_columns:
         table = c.table
-        source = scope.sources.get(table)
+        source = _find_table_source(scope, table)
```

Comment:
What triggered the need for this helper? We have logic dealing with pivots a few lines below. Can we instead adapt refactor that to also take `UNPIVOT` into account?

Reply:
@zeta9044 please don't mark unresolved comments as resolved.